### ISSUE TITLE
[Feature request] Jump to the previous VD

### ISSUE THREAD

**Issue by llinfeng** [FuPeiJiang/VD.ahk#16]
Now that we can jump to an arbitrary VD by number, it would help to have a way to jump back to where we came from :) Thx!

**Comment by Konfekt** [comment]
The function to switch to the last active desktop could perhaps be implement similar to [this](https://github.com/sdias/win-10-virtual-desktop-enhancer/pull/65) and [that](https://github.com/phazzzy/win-10-virtual-desktop-enhancer/commit/b1fb9255253092c394ace1f2f0afd6a6f62c56aa) commit...

**Comment by FuPeiJiang** [comment]
@Konfekt
what's the use case ? do you want alt+tab behavior but for virtual desktop ?
or do you want to store it in a history ? (instead of storing only the last active desktop)
```ahk
 #Include %A_LineFile%\..\..\VD.ahk 
 VD.RegisterDesktopNotifications() 
 VD.CurrentVirtualDesktopChanged:=Func("CurrentVirtualDesktopChanged") 
 CurrentVirtualDesktopChanged(desktopNum_Old, desktopNum_New) { 
     ToolTip % desktopNum_Old ", " desktopNum_New 
 } 
 ```
I will implement it using this code

then you will just do:
` VD.goToDesktopNum(VD.lastDesktopNum)`

**Comment by Konfekt** [comment]
Exactly, Single Alt+Tab for Desktops. Thank you very much, storing the last desktop in a variable on a desktop change and accessing it by a function bound to a keyboard shortcut, say Alt+Tab, achieves this.
